pkgs/by-name/fb/fbset/package.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, unstableGitUpdater
+, bison
+, flex
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "fbset";
+  version = "2.1";
+
+  src = fetchFromGitHub {
+    owner = "sudipm-mukherjee";
+    repo = "fbset";
+    rev = "debian/${finalAttrs.version}-33";
```

Comment:
Why not make `-33` part of `version`? (On the next version bump, the `-33` will change, right?)

Reply:
This package is a bit weird. The upstream repository is no longer online, I'm using the debian maintainer mirror of it.

The debian version scheme goes like `[upstream version]-[debian version for this specific upstream version]`.
Here this is the upstream version `2.1` and it's the 33rd revision of the package. I don't think the debian revision matters much to us, which is why I excluded it from the derivation's version.

Reply:
Ok.